src/initializers/postgres.ts

```diff
@@ -20,7 +20,9 @@ export default function postgres(config) {
       max: pgConfig.poolSize,
       ssl: pgConfig.useSsl ? pgConfig.sslConfig : undefined,
       idleTimeoutMillis: pgConfig.idleTimeoutMillis,
-      connectionTimeoutMillis: pgConfig.connectionTimeoutMillis
+      connectionTimeoutMillis: pgConfig.connectionTimeoutMillis,
+      statement_timeout: pgConfig.statementTimeout,
```

Comment:
Weird that they have different casing in the configuration

Reply:
Indeed - I didn't like it also 🤯